lib/lumberjack/client.rb

```diff
@@ -69,10 +67,9 @@ def initialize(opts={})
         :port => 0,
         :address => "127.0.0.1",
         :ssl_certificate => nil,
-        :window_size => 5000
       }.merge(opts)
       @host = @opts[:address]
-      @window_size = @opts[:window_size]
+      @window_size = 1
```

Comment:
What do you think if we make the `window_size` transparent, it would be defined by the number of elements send to the `#write` method and would follow what the forwarder is doing.